Jenkinsfile.testsuite

```diff
@@ -0,0 +1,55 @@
+@Library('ctsrd-jenkins-scripts') _
+
+// Set job properties:
+def jobProperties = [rateLimitBuilds([count: 1, durationName: 'hour', userBoost: true]),
+                     [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/CTSRD-CHERI/cheribsd/'],
+                     copyArtifactPermission('*'), // Downstream jobs need the kernels/disk images
```

Comment:
Not for this one they don't

Reply:
... and fixed whilst I was writing that :D

Reply:
copy-pasted and was fixed for run `#4`

Reply:
Jenkins testing is kinda awkward so I just keep using replay+jenkins pipeline unit stuff.